dotCMS/src/main/java/com/liferay/portal/ejb/CompanyManagerUtil.java

```diff
@@ -24,6 +24,8 @@
 
 import java.util.TimeZone;
 
+import com.dotcms.business.CloseDBIfOpened;
+import com.dotcms.business.WrapInTransaction;

```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'com.dotcms.business.WrapInTransaction'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AUselessImportCheck)